config/initializers/docker.rb

```diff
@@ -0,0 +1,13 @@
+# Docker specific development configuration
```

Comment:
moved this in an initializer because Rubocop was complaining that development.rb was becoming too big